nncf/quantization/algo.py

```diff
@@ -1090,6 +1090,27 @@ def _add_single_activation_quantizer(self, target_model: NNCFNetwork,
                 InsertionCommand(insertion_point, hook, OperationPriority.QUANTIZATION_PRIORITY))
         return quantizer_id, insertion_commands
 
+    def are_frozen_layers_allowed(self) -> bool:
+        for quantizer in self._weight_quantizers.values():
+            if not quantizer.quantizer_module_ref.num_bits == 8:
+                return False
+        if self.config.get('initializer', {}).get('precision', None) is not None:
```

Comment:
Use the value of `self._precision_init_args` instead of poking into the config, here and elsewhere.